packages/client-react-streaming/src/importErrors.test.tsx

```diff
@@ -0,0 +1,101 @@
+import assert from "node:assert";
```

Comment:
We don't have a test setup over in `@apollo/experimental-nextjs-app-support`, so we're testing this only over in this package.

Long-term it might be a good idea to mirror the test over there.

Reply:
But I just tried it out, just in case:
One works:
![image](https://github.com/apollographql/apollo-client-nextjs/assets/4282439/3b9dddfc-5073-405c-a9e4-0fb7eba646c0)

The other one doesn't yet - I'll get back on that.
![image](https://github.com/apollographql/apollo-client-nextjs/assets/4282439/b8f06efc-e968-47ba-840a-c3f7edfaa007)

Reply:
Looking better now!
![image](https://github.com/apollographql/apollo-client-nextjs/assets/4282439/3dbced09-4d02-4636-b77b-fce068ff4123)
![image](https://github.com/apollographql/apollo-client-nextjs/assets/4282439/55fdc543-5130-4658-b725-53fa4df99cba)